tests/unit_tests/dft/include/compute_out_of_place.hpp

```diff
@@ -60,9 +76,18 @@ int DFT_Test<precision, domain>::test_out_of_place_buffer() {
 
         {
             auto acc_bwd = bwd_buf.template get_host_access();
-            EXPECT_TRUE(check_equal_vector(acc_bwd.get_pointer(), out_host_ref.data(),
-                                           bwd_data.size(), abs_error_margin, rel_error_margin,
-                                           std::cout));
+            auto bwd_ptr = acc_bwd.get_pointer();
+            auto ref_iter = out_host_ref.begin();
+            const auto ref_row_stride = sizes.back();
+            const auto backward_row_stride = sizes.back();
```

Comment:
```suggestion
            const auto backward_row_stride = get_row_size<domain>(sizes);
```
(if opting for the other suggested changes to enabled 2D/3D batched r2c, I believe this is required)

Reply:
absolutely, thanks for saving me figuring that out! addressed in https://github.com/oneapi-src/oneMKL/pull/275/commits/14bfce1c072f440e22efaf30a7ee9db5252073f5